src/network/RTypeClient.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** RTypeClient
+*/
+
+#include <stdlib.h>
+#include "RTypeClient.hpp"
+
+RTypeClient::RTypeClient(std::string hostname, std::string port)
+: UDPConnection(hostname, port), gameEnd(false)
+{
+}
+
+void RTypeClient::engineInit()
+{
+    std::cout << "Init Game" << std::endl;
+    send("start");
+
+    uint16_t compNb = receiveUint16();
+    uint8_t compNameMaxSize = receiveUint8();
+
+    std::cout << "CompNb : " << (int)compNb << "." << std::endl;
+    std::cout << "CompNameMaxSize : " << (int)compNameMaxSize << "." << std::endl;
```

Comment:
Shouldnt there be a need of this compNameMaxSize element ?

Reply:
In the original protocol yes but since we send buffers, we don't need a max size, we juste receive the buffer and that's all

Reply:
We have to update the network protocol